content/doc/book/installing/_docker.adoc

```diff
@@ -78,14 +75,14 @@ directory on your local machine, which would typically be
 `/Users/<your-username>/jenkins` or `/home/<your-username>/jenkins`.
 <6> ( _Optional_ ) `/var/run/docker.sock` represents the Unix-based socket
 through which the Docker daemon listens on. This mapping allows the
-`jenkinsci/blueocean` container to communicate with the Docker daemon, which is
-required if the `jenkinsci/blueocean` container needs to instantiate other
+`jenkinsci/jenkins` container to communicate with the Docker daemon, which is
+required if the `jenkinsci/jenkins` container needs to instantiate other
 Docker containers. This option is necessary if you run declarative Pipelines
 whose syntax contains the <<pipeline/syntax#agent,`agent`>> section with the
 `docker` parameter - i.e. +
 `agent { docker { ... } }`. Read more about this on the
```

Comment:
It will not work by default. We would need to add documentation describing that they should install the Docker Pipeline plugin in the installation wizard

Reply:
Applied to other usages on this page

Reply:
There is a very specific reason the Blue Ocean image was used. The Docker Pipeline plugin is not enough to make these instructions work, the Docker image would need to have the Docker CLI installed.

There is effectively no simple way to get started with `jenkins/jenkins:lts` that doesn't require lots of agent configuration, or running workloads on the master.

The Blue Ocean image was curated to include a number of useful tools for getting started, which is why we used it as the basis for these steps.

Reply:
ftr https://github.com/jenkinsci/blueocean-plugin/blob/master/Dockerfile.build

Reply:
that one is to build BO imo, I would say the `jenkinsci/blueocean` image is built using https://github.com/jenkinsci/blueocean-plugin/blob/master/Dockerfile

Reply:
I am wondering if we should either modify the docs to remove the dependency against the `docker` CLI, or add the `docker` CLI to `jenkins/jenkins`.

Reply:
I don't see any references to installing `docker` in the Blue Ocean image.

Reply:
Also, why would you need to embed `docker` in the master image when you shouldn't be executing builds there? You can use `docker-compose` to set up a fixed number of jnlp-agents, or you could use Kubernetes or similar plugins for dynamic instances.

Reply:
> You'd need to install DinD on the agent image to allow it to spawn docker containers for a build using that feature

That's what I would expect is needed.  The build process relies on docker images instead of mandating that the user install many packages on their computer.